scapy/layers/bluetooth.py

```diff
@@ -362,6 +481,86 @@ class L2CAP_Connection_Parameter_Update_Response(Packet):
     fields_desc = [LEShortField("move_result", 0), ]
 
 
+class L2CAP_LE_Credit_Based_Connection_Request(Packet):
+    name = "L2CAP LE Credit Based Connection Request"
+    fields_desc = [LEShortField("spsm", 0),
+                   LEShortField("scid", 0),
+                   LEShortField("mtu", 0),
+                   LEShortField("mps", 0),
+                   LEShortField("initial credits", 0), ]
```

Comment:
This is incorrect and makes a test fail https://github.com/secdev/scapy/runs/4524985545?check_suite_focus=true#step:5:383

Reply:
Please launch the test manually on your dev environment.

Reply:
> Please launch the test manually on your dev environment.

Is there a way to check the code in my local environment? Is there only one way to ask for "approval" every time?